core/templates/components/button-directives/primary-link-button.component.html

```diff
@@ -0,0 +1,15 @@
+<a *ngIf="!isButton" class="oppia-primary-link-button"
```

Comment:
This class name matches the HTML tag used to represent the button, which could be confusing. Is it OK to use the HTML tag for CSS instead and drop this class? If not and we do need an inner class, then maybe call the class ".primary-button" or similar so it doesn't get confused with the html tag name.

Reply:
Renamed it